datasets/bkt/bkt.json

```diff
@@ -0,0 +1,367 @@
+          },
+          "peildatumAfleiding": {
+            "type": "string",
+            "format": "date"
+          },
+          "bgtObjectklasse": {
+            "type": "string"
+          },
+          "bkBgtObject": {
+            "type": "string"
+          },
+          "identificatieNamespace": {
+            "type": "string"
+          },
+          "identificatieLokaalid": {
+            "type": "string"
+          },
+          "objectbegintijd": {
+            "type": "string",
+            "format": "date"
+          },
+          "objecteindtijd": {
+            "type": "string",
+            "format": "date"
+          },
+          "tijdstipregistratie": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "eindregistratie": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "lvPublicatiedatum": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "bronhouder": {
+            "type": "string"
+          },
+          "inonderzoek": {
+            "type": "boolean"
+          },
+          "relatievehoogteligging": {
+            "type": "string"
+          },
+          "optalud": {
+            "type": "boolean"
+          },
+          "bgtStatus": {
+            "type": "string"
+          },
+          "plusStatus": {
+            "type": "string"
+          },
+          "bgtFunctie": {
+            "type": "string"
+          },
+          "plusFunctie": {
+            "type": "string"
+          },
+          "bgtFysiekvoorkomen": {
+            "type": "string"
+          },
+          "plusFysiekvoorkomen": {
+            "type": "string"
+          },
+          "bgttype": {
+            "type": "string"
+          },
+          "plustype": {
+            "type": "string"
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/MultiPolygon.json"
+          },
+          "aggOppIntersectie": {
+            "type": "number"
+          },
+          "aggFractie": {
+            "type": "number"
+          },
+          "aggAantalPercelen": {
+            "type": "integer"
+          },
+          "aggGerelateerdePercelen": {
+            "type": "string"
+          },
+          "aggGrootstePerceel": {
+            "type": "string"
+          },
+          "aggAardZakelijkRechtCode": {
+            "type": "string"
+          },
+          "aggAardZakelijkRechtOmschr": {
+            "type": "string"
+          },
+          "aggIndicatieBelastRecht": {
+            "type": "boolean"
+          },
+          "aggBelastDoorZrtCode": {
+            "type": "string"
+          },
+          "aggBelastDoorZrtOmschr": {
+            "type": "string"
+          },
+          "aggIndBetrokkenBijSplitsing": {
+            "type": "boolean"
+          },
+          "kaartUrl": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    {
+      "id": "bgt",
```

Comment:
Is dit de 'gewone' bgt? Misschien moeten we die dan eerst snel importeren zodat het gewone relaties kunnen worden.